lib/src/flutter/runners/gherkin_integration_test_runner.dart

```diff
@@ -283,28 +296,43 @@ abstract class GherkinIntegrationTestRunner {
       multiLineStrings,
     );
 
-    final result = await executable.step.run(
-      dependencies.world,
-      reporter,
-      configuration.defaultTimeout,
-      parameters,
-    );
+    StepResult? result;
 
+    if (hasToSkip) {
+      result = new StepResult(
+          0, StepExecutionResult.skipped, "Previous step(s) failed.");
+    } else {
+      for (int i = 0; i < this.configuration.stepMaxRetries; i++) {
```

Comment:
When did `stepMaxRetries` get added to the configuration? Was this in a previous PR?

Reply:
This has now been added in the dart_gherkin package. I have also created a PR there. That is why it is required for dart_gherkin to be updated before this package.

Reply:
Make sense!